meta-lmp-base/recipes-security/optee/optee-os-tadevkit_3.17.0.bb

```diff
@@ -1,12 +1,17 @@
+DEFAULT_PREFERENCE = "-1"
```

Comment:
With default preference -1 we still require a preferred_version set somewhere. We can probably have it under meta-lmp-bsp (machine custom), done for all machines there.

Users just using meta-lmp-base would not be affected by them.

Reply:
Wonder if this could be conditional based on the preferred provider, something to experiment with.